src/types/time.rs

```diff
@@ -84,10 +196,10 @@ mod test {
         };
 
         ts_vec.push(make_datetime(10_000, 0)); //January 1, 1970 2:46:40 AM
-        ts_vec.push(make_datetime(10_000, 1000)); //January 1, 1970 2:46:40 AM (and one microsecond)
+                                               // ts_vec.push(make_datetime(10_000, 1000)); //January 1, 1970 2:46:40 AM (and one microsecond)
```

Comment:
Why ?

Reply:
Sorry leftover from when I tried to rewrite the `OffsetDateTime` implementation. Uncommented the code.

Reply:
Someone else fixes it here: #1374